scripts/ssb-web/uninstall

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+sudo rm -rf /var/www/backend
+sudo rm -rf /var/www/sbot
+sudo rm -rf /usr/lib/node_modules
+sudo rm -rf /etc/nginx/site-path-enabled/ssb-web.conf
+
+sudo npm uninstall -g --unsafe-perm \
+                 ssb-keys \
+                 ssb-client \
+                 ssb-feed \
+                 pull-stream
+sudo apt-get remove -y php-fpm socat                 
```

Comment:
This script should change the perms back, and remove /var/www/.ssb. /usr/local/bin/sbot should be removed as well, and so should $BASEDIR/tmp/ssb-web-pi, maybe even at the end of the of the install script.